Dimer_scf_vs_radius.py

```diff
@@ -0,0 +1,42 @@
+from wrap_pyscf import *
+
+import pickle
+
+scf_guess = {}
+
+filename = "scf_record"
+
+if "Dimer equill" not in scf_guess:
+    funct_grps = [[1,0,2], [3,4,5]]
```

Comment:
I recommend getting in the habit of reducing code duplication whenever you can, it'll save you a lot of headaches when you have to go back to the code at a later point. In this case, the `funct_grps` variable is the same in each `if` statement so factor it out and set it before the if statement. You also could use a loop to go over the various geometries (loop from 0 to 7, your description is `2*i +2` where `i` is the loop index and your file is just `i`).